src/stdlib_linalg_lapack_aux.fypp

```diff
@@ -321,7 +321,7 @@ module stdlib_linalg_lapack_aux
      end function stdlib_ilauplo
 
 
-     pure integer(ilp) function stdlib_iparmq( ispec, name, opts, n, ilo, ihi, lwork )
```

Comment:
@cyrilgandon please be careful changing the BLAS/LAPACK backend API. I would believe we don't want it modified, so it is always compatible with the library's standard interfaces

Reply:
I agree with @perazz : API of BLAS/LAPACK should not be modified (and I would extend that to all procedures of `stdlib`). Such API changes should have their own PR.

Reply:
Oh yeah, ,thank you for pointing that, for a minute I believe iparmq was an internal function, my bad.
BLAS and LAPACK generates most of the unused-dummy warnings, do you have a preferred way to deal with those?

Reply:
There is not much we can do about it as far as I can tell. There may be ways some those warnings could be suppressed (i.e., force their initialization, at least if they are `intent(out)/intent(inout)`, but maybe a better strategy would be to flag the BLAS/LAPACK backend sources with `-Wnounused-...` as pointed out by @Romendakil